packages/x-date-pickers/src/locales/beBY.ts

```diff
@@ -45,7 +45,7 @@ const beBYPickers: Partial<PickersLocaleText<any>> = {
 
   // Clock labels
   clockLabelText: (view, time, utils, formattedTime) =>
-    `Абярыце ${views[view]}. ${!formattedTime && (time === null || !utils.isValid(time)) ? 'Час не абраны' : `Абраны час ${formattedTime ?? utils.format(time, 'fullTime')}`}`,
+    `Абярыце ${views[view]}. ${!formattedTime && (time === null || !utils.isValid(time)) ? 'Час не абраны' : `Абраны час ${formattedTime ?? utils.format(time!, 'fullTime')}`}`,
```

Comment:
It's also really not pretty we have to do this. 🙈 
Thankfully this is going away once we merge the refactor of these cases. 🤞

Reply:
Yes :laughing: